source/import-export.txt

```diff
@@ -127,10 +127,23 @@ Export Data from a Collection
 -----------------------------
 
 |compass| can export data from a collection as either a
-**JSON** or **CSV** file. If you specify a query in the
+**JSON** or **CSV** file. If you specify a
+:ref:`filter <query-bar-filter>` in the
 :ref:`query bar <compass-query-bar>` prior to export, |compass-short|
 can optionally only export documents which match the specified query.
 
+Behavior
+~~~~~~~~
+
+While it is possible to exclude documents by using a query filter,
+it is not possible to re-shape exported documents with a
+:ref:`project <query-bar-project>` document. Even when you specify a
+project option in the query, |compass-short| still exports the entire
```

Comment:
I'd format the word "project" somehow, maybe with double-backticks, just because "project" is such an overloaded word.